rust/src/container.rs

```diff
@@ -35,36 +35,36 @@ pub async fn entrypoint(args: &[&str]) -> Result<i32> {
     Ok(0)
 }
 
-#[derive(Debug, StructOpt)]
+#[derive(Debug, Parser)]
 struct ContainerEncapsulateOpts {
-    #[structopt(long)]
-    #[structopt(parse(try_from_str = ostree_ext::cli::parse_repo))]
-    repo: ostree::Repo,
+    #[clap(long)]
+    #[clap(value_parser)]
+    repo: Utf8PathBuf,
```

Comment:
Note: I had to move this to a plain path string, because `ostree::Repo` isn't `Sync` (and `clap` doesn't seem to like that).

Reply:
Hmm that's mildly unfortunate.  Presumably clap wants the whole options struct to be `Sync` for some reason?

Or hmm, looks like the bound may be here https://github.com/clap-rs/clap/blob/b619699539fb18b2d62f11d27fafebf7a85cafef/src/parser/matches/arg_matches.rs#L117 ?

But it's unclear why.

Reply:
Yes to both.
My understanding is that those trait bounds may be coming from `Any` requirements, likely these: https://doc.rust-lang.org/std/any/trait.Any.html#impl-2